translations/default/AdminCatalogFeature.xlf

```diff
@@ -867,6 +867,73 @@
         <target>Default delivery time: [1]%delivery_time% - %oos_delivery_time%[/1] [2]Edit delivery time[/2]</target>
         <note>Line: 96</note>
       </trans-unit>
+      <trans-unit id="6adf97f83acf6453d4a6a4b1070f3754">
+        <source>None</source>
+        <target>None</target>
+        <note>Line: 95</note>
+      </trans-unit>
+      <trans-unit id="16599dbd0bf238cbae4090fe1db04145">
+        <source>Specific delivery time to this product</source>
+        <target>Specific delivery time to this product</target>
+        <note>Line: 104</note>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="src/Core/Form/ChoiceProvider/OutOfStockTypeChoiceProvider.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="989d17fa8f3081cc0d66838c7504c0ef">
+        <source>Use default behavior</source>
+        <target>Use default behavior</target>
+        <note>Line: 71</note>
+      </trans-unit>
+      <trans-unit id="119a7c88cf8f1c685c57981abff26f43">
+        <source>Allow orders</source>
+        <target>Allow orders</target>
+        <note>Line: 66</note>
+      </trans-unit>
+      <trans-unit id="f072da215c080d985d44361e96a3cf60">
+        <source>Deny orders</source>
+        <target>Deny orders</target>
+        <note>Line: 67</note>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="src/Core/Form/ChoiceProvider/PackStockTypeChoiceProvider.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="175db77b6b3ecd9633e85b5efc28bbc1">
+        <source>Decrement pack only.</source>
+        <target>Decrement pack only.</target>
+        <note>Line: 82</note>
+      </trans-unit>
+      <trans-unit id="1ba020dea4ce06fa139a471db21f41a7">
+        <source>Decrement products in pack only.</source>
+        <target>Decrement products in pack only.</target>
+        <note>Line: 83</note>
+      </trans-unit>
+      <trans-unit id="61b912928e920c280798dc648d4068fe">
+        <source>Decrement both.</source>
+        <target>Decrement both.</target>
+        <note>Line: 84</note>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="src/Core/Form/ChoiceProvider/ProductConditionChoiceProvider.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="03c2e7e41ffc181a4e84080b4710e81e">
+        <source>New</source>
```

Comment:
Yep exactly, maybe we should use the one from the shop then no? Do you know if it's still `New` in FO?
I believe there's a confusion between `new = neuf` and `new = récent` but I don't know how it was solved

Reply:
No we should use the Admin string, I just wanted to double check. 

Today we have:

Product page, back office =
‘New’ in Admin.Catalog.Feature
‘Used’ in Admin.Catalog.Feature
‘Refurbished’ in Admin.Catalog.Feature

New label in the product listing, front office + New in the product page, front office (condition) =

‘New’ in Shop.Theme.Global

The solution is to have:

Product page, back office :

‘New’ in Admin.Catalog.Feature
‘Used’ in Admin.Catalog.Feature
‘Refurbished’ in Admin.Catalog.Feature

New label in the product listing, front office:

‘New’ in Shop.Theme.Global

New in the product page, front office (condition):

‘New’ in Shop.Theme.Catalog
‘Used’ in Shop.Theme.Catalog
‘Refurbished’ in Shop.Theme.Catalog

Reply:
Guys not again please :D

Thats why I said that I don't like these hackjob solutions to change domain from something like Shop.Theme.Catalog to Shop.Theme.Global. Good luck finding what is what. 👍

Reply:
It's the only way I see to solve the translation issue we have right now. If you have a better idea, I would love to hear it. ☺️

Reply:
We could also add a comment in Crowdin with a screenshot to make sure translators have enough context.

Reply:
I proposed on Slack (maybe you weren't in the conversation, but Lea?), that it would be better if we start using more specific domains.

For example:
Shop.Product.Label
Shop.Product.Condition

Then there would be no issue distinguishing what "New" means. :-)

Reply:
Yes sorry, I remember now, it was me. I don't think it's a good idea to create a new domain for only a string. @eternoendless and @MatShir also mentioned an issue related to themes.